.travis.yml

```diff
@@ -16,8 +16,8 @@ before_script:
   - ./manage.py migrate
 
 script:
-  - python3.6 -m pytest -v vulnerabilities/tests/test_scrapers.py vulnerabilities/tests/test_api_data.py
-  - ./manage.py test vulnerabilities/tests
+  - python3.6 -m pytest -v importers/tests/test_scrapers.py api/tests/test_cve_search.py
```

Comment:
Since you are changing this, what about running `pytest -vvs` only and adding a proper entry in setup.cfg to guide the discovery?

Reply:
I thought about changing this since I don't like passing files as arguments here either. But I don't know how to do this without reading some docs and I figured it will be changed as part of #127 anyway.